packages/loader/container-loader/src/container.ts

```diff
@@ -188,6 +188,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         loader: Loader,
         request: IRequest,
         resolvedUrl: IFluidResolvedUrl,
+        pendingOps?,
```

Comment:
same here (and possibly in other places)

Reply:
naming: From usage perspective, this is "unsaved local state", i.e. I do not think host should operate with terms like ops. That's our internal implementation detail

Reply:
Does it need to be serialized? I think that will cause problems with local metadata on sequence ops.

Reply:
Yes, it needs to be serializable. iFrame that hosts this container will be destroyed, and new iFrame will be created in some future point in time. iFrame is the unit of JS execution (i.e. new JS context), the only thing that we can get out / sent to iFrame is strings.